system-contract-dapp-playground/src/api/hedera/tokenTransfer-interactions/index.ts

```diff
@@ -95,7 +98,7 @@ export const transferFungibleTokens = async (
     });
   }
   if (!sanitizeErr) {
-    amounts.some((amount) => {
+    amounts.slice(1).some((amount) => {
```

Comment:
Why are you taking only the 1st element?

Reply:
Oh the `amounts.slice(1)` actually helps skipping the first element of the array in the loop. The reason is that the initial item in the `amounts` array represents the `totalInputAmount` multiplied by -1, ensuring that the sum of the amounts can equal 0. For instance, if a client inputs 2 amounts, 100 and 200, the `amounts` array would be [-300, 100, 200]. This is why, when sanitizing the parameter, we only need to check on the amounts provided by the client.

Reply:
Okay, a short comment with this context on top of this line to help readers